### DISCUSSION TITLE
A simple wish for 1.14

### DISCUSSION THREAD

**Discussion by flarediablo** [flareteam/flare-game#918] [Ideas]
Make the items for the Brute (Warrior) and Scout (Rogue) look different. Both when equipped and the icons in the inventory.

Currently both sets look the same, with the Adept (Mage) having its unique cloth armor and staff mage stuff having a different look.

Warrior = heavy plate to heavy chain
Rogue = light chain to leather
Mage = minimal leather to cloth

Don't forget about the headgear: helms, hats, crowns, hoods etc.

**Comment by Danimal696** [comment]
Hi, that´s the result of having just a handful of armor models and having to make do with them. Hopefully more armor models will be added in the future with more unique looks and types.

**Comment by WithinAmnesia** [comment]
Tint the armour sprite sheet for the armour. Try https://www9.lunapic.com/editor/?action=tint Lunapic [filters -> colour tint]. You can use Lunapic for free online if you don't have other stuff like Paint.NET https://www.getpaint.net/ (or paid ones like photoshop). Try different stuff; I like to use about 15% but its up to you. Also try like contrast, brightness, sharpness, saturation, hue etc. these are 2.5D assets so they can be conventionally image edited and tweaked easier than most 3D models if you want moderate unique different and quick results. It is not the same as 3D models but it works well enough to get the feeling of variety and breath new life into older assets with low input high output efficient work flow.
